firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -24,6 +24,9 @@ import com.google.firebase.ktx.Firebase
 import com.google.firebase.ktx.app
 
 class FirebaseSessions internal constructor(firebaseApp: FirebaseApp) {
+
+  private val sessionGenerator = SessionGenerator(true)
```

Comment:
Can you write it with a named param like `SessionGenerator(collectEvents = true)`? It will explain the meaning of the boolean literal.

Reply:
Yes I'll do this everywhere - didn't realize it was possible!